src/nmodl/lex.lpp

```diff
@@ -19,14 +19,75 @@
 
 
 #include "modl.h"
-#include "parse1.h"
+#include "parse1.hpp"
 
 int lexcontext = 0;
 extern int Getc();
-extern int unGetc();
-extern int unput();
+extern int unGetc(int);
+extern int unput(int);
+static int input();
+static int yyinput();
+//TODO - does this actually work?
+int newinput() {
```

Comment:
to clarify - anything needs to be checked here?

Reply:
No, it does work. Will remove the TODO